server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SonarQubeMonitor.java

```diff
@@ -19,30 +19,45 @@
  */
 package org.sonar.server.platform.monitoring;
 
+import com.google.common.base.Function;
```

Comment:
The goal of the ticket is to returned enabled identity providers in api/system/info response, so no, it's not expected. But have you at least one enabled identity provider ? Is it displayed in the login page ? To be fully enabled, the configuration need also to be feed (at least for GitHub and Bitbucket plugin)